framework/graphics/fps_info.cpp

```diff
@@ -68,17 +44,24 @@ FpsInfo::FpsInfo(uint64_t               measurement_start_frame,
                  const std::string_view measurement_file_name,
                  bool                   quit_after_frame,
                  uint64_t               quit_frame) :
-    measurement_start_frame_(measurement_start_frame),
-    measurement_end_frame_(measurement_end_frame), measurement_start_time_(0), measurement_end_time_(0),
-    has_measurement_range_(has_measurement_range), quit_after_range_(quit_after_range),
-    flush_measurement_range_(flush_measurement_range), flush_inside_measurement_range_(flush_inside_measurement_range),
-    started_measurement_(false), ended_measurement_(false), frame_start_time_(0), frame_durations_(),
-    measurement_file_name_(measurement_file_name), preload_measurement_range_(preload_measurement_range),
+    start_time_(0),
+    replay_start_time_(0), replay_end_time_(0), measurement_start_time_(0), measurement_end_time_(0),
+    measurement_start_boot_time_(0), measurement_end_boot_time_(0), measurement_start_process_time_(0),
+    measurement_end_process_time_(0), has_measurement_range_(has_measurement_range), replay_start_frame_(0),
+    measurement_start_frame_(measurement_start_frame), measurement_end_frame_(measurement_end_frame),
+    quit_after_range_(quit_after_range), flush_measurement_range_(flush_measurement_range),
+    flush_inside_measurement_range_(flush_inside_measurement_range), started_measurement_(false),
+    ended_measurement_(false), preload_measurement_range_(preload_measurement_range),
+    measurement_file_name_(measurement_file_name), frame_start_time_(0), frame_durations_(),
     quit_after_frame_(quit_after_frame), quit_frame_(quit_frame)
 {
     if (has_measurement_range_)
     {
-        GFXRECON_ASSERT(!measurement_file_name_.empty());
+        if (util::filepath::IsFile(measurement_file_name_))
+        {
+            // To avoid thinking an ancient file is the result of this run
+            std::remove(measurement_file_name_.c_str());
```

Comment:
I would use GFXRECON_LOG_WARNING_ONCE to note that it is physically replacing the old file.